README.md

```diff
@@ -1318,13 +1318,56 @@ ingress:
 * Description: If you use an image that is not public. then you can create dockerconfigjson secrets on your cluster and
   reference them here.
 
+### Elastic
+
+#### `elastic.enabled`
+
+* Required: false
+* Default: `false`
+* Description: Enable or disable elastic search subsystem
+> [!IMPORTANT]  
+> Alfresco can only use one search subsystem at a time, if you still have SOLR enabled (which you -shouldn't- when enabling this option),
+> SOLR will take precedence in the alfresco configmap.
```

Comment:
You could also consider making the helm chart fail when both elastic.enabled & solr.enabled are set, using the `fail` function: https://helm.sh/docs/chart_template_guide/function_list/#fail

Reply:
At the top of a file is fine